lib/docker.ts

```diff
@@ -92,6 +101,17 @@ class Docker {
     }
   }
 
+  public async size(): Promise<number | undefined> {
+    try {
```

Comment:
why do you need to wrap it with size? not sure if it should be a safe function

Reply:
Errors are ignored for backwards compatibility with the expected behaviour as being reflected by many tests.
I renamed the function to `sizeSafe` to better expose this,